html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="" method="post">
+      <label for="email">Email</label><br />
+      <input type="email" id="email" name="email" /><br />
+
+      <label for="simple">Why is coding cool?</label><br />
+      <select id="simple" name="simple">
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
+      <p>
+        <label>Favorite Language (choose one)</label>
```

Comment:
Since this `label` isn't associated with any form control element, it shouldn't be a `label`. You should just have the text inside the `p` tags